GLOSSARY.md

```diff
@@ -0,0 +1,47 @@
+# GitOps Glossary
+
+This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other supporting documents in this repository.
+
+- ## Break Glass
+
+    The temporary suspension of GitOps principles, often accomplished by pausing automated _Reconciliation_.
+    While these principles apply to typical operations, it may at times be necessary to temporarily pause reconciliation, for example during incident management activities.
+    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the system with the updated declarations.
+    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
+
+- ## Continuous
+
+    By "continuous" we adopt the industry standard to mean that _Reconciliation_ continues to happen, not that it must be instantaneous.
+
+- ## Declarative Description
+
+    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating configuration (the desired state) from the implementation (commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
+
+- ## Desired State
+
+    The aggregate of all configuration data for a system form its _Desired State_ which is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable.
+
+- ## Drift
+
+    When a system's _Actual State_ changes for any reason other than its versioned _Desired State_ declarations having changed, we say that the system has drifted from its _Desired State_.
+
+- ## Reconciliation
+
+    The process of ensuring that the _Actual State_ of a sytem matches its versioned _Desired State_ declarations.
```

Comment:
`s/sytem/system/g`

Reply:
@RedbackThomson would you please DM me your email for co-authorship purposes in the commit?

Reply:
For now I noted your username in the commit, and added a to-do item to the PR description to add co-author credits to this PR for everyone who participates. Thank you!

Reply:
Got your email. Amended the commit to include your co-authored-by info 👍